extra-multimedia/kodi/spec

```diff
@@ -1,5 +1,8 @@
-VER=18.5
+VER=18.9
 CODE=Leia
-SRCTBL="https://github.com/xbmc/xbmc/archive/$VER-$CODE.tar.gz"
-CHKSUM="sha256::108979df8b41ab4168f5cdc0233f46e38767eda5921f4ccae16584e98d0d6b29"
-REL=1
+SRCS="tbl::rename=kodi::https://github.com/xbmc/xbmc/archive/$VER-$CODE.tar.gz \
+      tbl::rename=kodi-res::https://github.com/xbmc/repo-resources/archive/4a733e3f10ee41f0d639376b11706d916974ae94.tar.gz"
```

Comment:
It's probably better to use a Git source here, since you are pulling a Git snapshot anyways.